### PR TITLE
Fixing python-manilaclient job

### PR DESCRIPTION
**Pull request by jsanemet**
Getting the job to install test dependencies in order for it to work. Proof in here: https://code.engineering.redhat.com/gerrit/c/python-manilaclient/+/421391

I also removed a couple of useless vars from the job I fixed before.

### CODE REVIEW

File: znoyder/config.yml [hunk 1]
```diff
@@ -808,8 +810,6 @@ override:
       'osp-rpm-py39':
         voting: true
         vars:
-          rhos_release_args: '17.0'
-          rhos_release_extra_repos: rhelosp-17.0-trunk-brew
```
Comment (cescgina) [inline znoyder/config.yml:812]: Have we tested the tripleo-ansible without this variables? Or are we sure that this extra repos are not needed with rpm jobs?

**Comment by jsanemet** [comment]
They are defined for all jobs at the top of the config, so this here is redundant. On any case, once we launch the Znoyder Jenkins we can take a look and see that there is no change for this job afterwards.

**Comment by cescgina** [comment]
That's true, I had missed that, thanks!

**Review by cescgina — APPROVED** [review]
LGTM

**Pull request merged**